.github/workflows/checks.yml

```diff
@@ -25,15 +25,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        command: ["prettier", "types", "lint", "test"]
```

Comment:
This test check should be added too

Reply:
And it should launch interface package tests

Reply:
oh so that's why the `test` is not running during the CI check?

Reply:
Right

Reply:
there's already a workflow for the interface e2e tests btw so probably should remove this `test` command from here

Reply:
I thought there are some unit tests and this task runs these sets.

Reply:
yea there were some (https://github.com/privacy-scaling-explorations/maci-platform/blob/fe977d7ff85959c9fe579a3dbee02823a35af1cb/src/lib/eas/__test__/eas.test.ts) - testing creating an eas attestation, not sure we want to keep them?

Reply:
I agree that we should build some unit tests for interface, but not in this PR, else it would be too large. 
Maybe open an issue to build unit tests and add the check(test) back, but remove it in this PR?

Reply:
Ok, let's remove this check because there are no tests right no

Reply:
sounds like a good plan